src/plugins/intel_npu/tools/protopipe/README.md

```diff
@@ -515,67 +515,79 @@ Iteration <number>:
+    ```
+- Init OpenVINO enviroment
+    ```
+     source "<path-to-openvino-install-dir>/setupvars.sh"
+    ```
+2. Build OpenCV
+- Clone OpenCV repo:
     ```
     git clone https://github.com/opencv/opencv
-    cd opencv && git checkout 78195bc3df
+    cd opencv && git checkout 3919f33e21
     ```
-2. Build OpenCV G-API:
-	```
-    mkdir -p build && cd build
+- Build OpenCV G-API:
+    ```
+    mkdir -p "build" && cd "build"
+    cmake ../ -DBUILD_LIST=gapi          \
+              -DCMAKE_BUILD_TYPE=Release \
+              -DWITH_OPENVINO=ON
+    cmake --build . --config Release --target opencv_gapi --parallel
+    ```
+    If ONNX support is needed build OpenCV G-API with ONNX support:
+    ```
+    mkdir -p "build" && cd "build"
     cmake ../ -DBUILD_LIST=gapi                             \
               -DCMAKE_BUILD_TYPE=Release                    \
               -DWITH_OPENVINO=ON                            \
-              -DOpenVINO_DIR=<path-to-openvino-install-dir> \
               -DWITH_ONNX=ON                                \
               -DORT_INSTALL_DIR=<path-to-onnxrt-install-dir>
     cmake --build . --config Release --target opencv_gapi --parallel
-	```
-### In-plugin build
-
-1. Clone and build [OpenVINO](https://github.com/openvinotoolkit/openvino) from sources
-2. Build OpenCV G-API with OpenVINO / ONNXRT support
-3. Clone `npu-plugin` repository
-	```
-	git clone https://github.com/openvinotoolkit/npu_plugin
-	git submodule update --init --recursive
-	```
-4. Build `Protopipe` as part of the `npu-plugin` build:
-	```
-	mkdir build && cd build
-	cmake ../ -DOpenCV_DIR=<path-to-opencv-build> -DOpenVINODeveloperPackage_DIR=<path-to-openvino-build>
-	cmake --build . --config Release --target protopipe --parallel
-	```
-
+    ```
+#### Build Protopipe inside openVino
+1. Build protopipe
+    ```
+    cd <path-to-OpenVINO-root-dir>
+    mkdir "src/plugins/intel_npu/tools/protopipe/build" && cd "src/plugins/intel_npu/tools/protopipe/build"
+    cmake ../ -DOpenCV_DIR=<path-to-opencv-build-dir> -DCMAKE_BUILD_TYPE=Release
+    cmake --build . --target protopipe --parallel --config Release
+    ```
 ### Standalone build
 1. Build `yaml-cpp`
-	```
-	mkdir -p yaml-cpp_build cd && yaml-cpp_build
-	cmake ../<npu-plugin>/thirdparty/yaml-cpp -DCMAKE_INSTALL_PREFIX=install
-	cmake --build . --config Release --target install --parallel
-	```
+   ```
+   mkdir -p "yaml-cpp_build" && cd "yaml-cpp_build"
+   cmake <openvino-dir>/src/plugins/intel_npu/thirdparty/yaml-cpp -DCMAKE_INSTALL_PREFIX=install
+   cmake --build . --config Release --target install --parallel
+   ```
 2. Build `gflags`
-	```
-	git clone https://github.com/gflags/gflags
-	cd gflags
-	mkdir -p gflags_build cd && gflags_build
-	cmake ../ -DCMAKE_INSTALL_PREFIX=install
-	cmake --build . --config Release --target install --parallel
-	```
+   ```
+   git clone https://github.com/gflags/gflags
+   cd gflags
+   mkdir -p "gflags_build" && cd "gflags_build"
```

Comment:
`mkdir -p "..."` also does some weird stuff for me (on Windows). It creates `-p` folder for me... Let me check how this could be replaced

Reply:
Well the commands are for Bourne Shell (sh). sh commands do different thinks on windows(cmd or powershell).
For powershell the alternative is "New-Item -ItemType Directory -Force -Path C:\Path\That\May\Or\May\Not\Exist"
for cmd the alternative is "mkdir" without the "-p" but "mkdir" in cmd is unable to handle the directory separator "/"